config/management/operational/src/validator_config.rs

```diff
@@ -184,3 +185,18 @@ fn decode_address(raw_address: RawNetworkAddress) -> Result<NetworkAddress, Erro
         .collect::<Vec<_>>();
     Ok(NetworkAddress::try_from(protocols).unwrap())
 }
+
+#[derive(Debug, StructOpt)]
+pub struct ValidatorConfig {
+    #[structopt(long, help = "JSON-RPC Endpoint (e.g. http://localhost:8080")]
+    host: String,
```

Comment:
I think we should rename this to json_server, sinceh ost is kinda ambiguous. That's what I'm naming it in the config and will update all places after I make my hopeful progress.

Reply:
Okay, I'll update it here and assume that you'll catch the rest of the places 😄